R/use_import_from.R

```diff
@@ -29,8 +29,7 @@ use_import_from <- function(package, fun, load = is_interactive()) {
   check_uses_roxygen("use_import_from()")
   check_installed(package)
   check_has_package_doc("use_import_from()")
-
-  purrr::walk2(package, fun, check_fun_exists)
+  check_functions_exist(package, fun)
```

Comment:
This isn't strictly necessary, but it makes the mock a little simpler and it's consistent with the other checks